helm/credentiald/Chart.yaml

```diff
@@ -3,5 +3,5 @@ name: credentiald
 description: >-
   credentiald manages credentials for cloud environments.
 home: https://github.com/giantswarm/credentiald
-version: [[ .Version ]]
-appVersion: [[ .AppVersion ]]
+version: 100.0.0
+appVersion: 100.0.0
```

Comment:
The test code needs to be removed. Also this architect templating is so annoying.

Reply:
Me either, Can we maybe escalate this to team-biscuit ?

Reply:
Yes will do. There is also the idea to have a VERSION file and not need this templating.

https://gigantic.slack.com/archives/C04TGHDEF/p1591257038015600

I'll also check how that is going.